lib/newProfile.ml

```diff
@@ -131,6 +132,11 @@ let make_mem_diff ~(mstart:Gc.stat) ~(mend:Gc.stat) =
   let pp tdiff = `String (Format.sprintf "%.3G w" tdiff) in
   [("major",pp major); ("minor", pp minor)]
 
+let make_instr_diff ~istart ~iend =
+  match System.instructions_between ~c_start:istart ~c_end:iend with
+  | Ok count -> [("instr", `Intlit (Int64.to_string count))]
```

Comment:
There is probably an appropriate field we could use, instead of sticking the instruction count in the "args" field that is used for non-standard things.

Reply:
What makes you think that?

Reply:
I searched in the documentation and found a reference to `tidelta` and `ticount` fields that seem relevant. But that's on the last page of the document, and I am not sure if it is really part of the format, and if it is supported by tools.

Reply:
We can try using that if you want, generate a trace with it and try putting it in perfetto (or something else if you know a better viewer)